main.py

```diff
@@ -75,19 +65,28 @@ async def run(
     url: str = 'https://maps-demo-git-katamartin-benchmarking-carbonplan.vercel.app/',
     playwright_python_version: str | None = None,
     provider_name: str | None = None,
-    screenshot_dir: pathlib.Path,
-    trace_dir: pathlib.Path,
+    trace_dir: upath.UPath,
+    action: str | None = None,
+    zoom_level: int | None = None,
 ):
     # Launch browser and create new page
-    browser = await playwright.chromium.launch()
+    chrome_args = [
+        '--enable-features=Vulkan,UseSkiaRenderer',
+        '--use-vulkan=swiftshader',
+        '--enable-unsafe-webgpu',
+        '--disable-vulkan-fallback-to-gl-for-testing',
+        '--dignore-gpu-blocklist',
+        '--use-angle=vulkan',
+    ]
+    browser = await playwright.chromium.launch(args=chrome_args)
```

Comment:
i was able to ensure GPU rasterization. confirmed this by launching chromium with `headless=False` and visiting `chrome://gpu` page. i get the following, 



```
Graphics Feature Status
Canvas: Hardware accelerated
Canvas out-of-process rasterization: Disabled
Direct Rendering Display Compositor: Disabled
Compositing: Hardware accelerated
Multiple Raster Threads: Enabled
OpenGL: Enabled
Rasterization: Hardware accelerated
Raw Draw: Disabled
Skia Graphite: Disabled
Video Decode: Hardware accelerated
Video Encode: Software only. Hardware acceleration disabled
Vulkan: Enabled
WebGL: Hardware accelerated
WebGL2: Hardware accelerated
WebGPU: Software only, hardware acceleration unavailable
```

Reply:
@andersy005 can you please clarify the source of this report? is this local or on an ec2 instance?

Reply:
this is from my local linux machine.

Reply:
unfortunately, i'm getting different results on an instance provisioned via the jupyterhub ![testing](https://github.com/carbonplan/benchmark-maps/assets/13301940/d216df83-a530-489e-b247-11c2a86722bd)